.circleci/config.yml

```diff
@@ -0,0 +1,91 @@
+version: 2.1
+orbs:
+  node: circleci/node@5
+
+aliases:
+  - &release-branches
+    - master
+    - develop
+    - /^hotfix\//
+
+executors:
+  default-executor:
+    docker:
+      - image: cimg/node:16.18.1 # Consider switching to LTS when scratch-audio supports that version
+    working_directory: ~/project
+    resource_class: medium
+
+commands:
+  test:
+    steps:
+      - run:
+          name: Test
+          command: npm run test
```

Comment:
Looking at `node/install-packages` source, it should also be using whatever node/npm is available in the executor, so I don't think it will be using a different version. (However, things could be different if we were using `node/install`, which we aren't.)

But there is a `node/test` job that simplifies some of this logic. I pushed ac0f98c43 with this change. There is one downside that I see. I don't see how I can persist the workspace from the `node/test` job and use it in the `deploy` job. That means I need to checkout and install again in the deploy step. If I understand this correctly, it is using a cache for this, but it does end up taking about 12 seconds longer. I'm torn. This new way feels simpler, but is it worth it? What do you think @meyerhot95?

Reply:
@aoneill01 sometimes the `checkout` and `install` is faster is faster than the `persist` as was the case with scratch-resources but this seems to be the opposite. I would recommend going with the original version.

Reply:
Probably this is because of all the non node_modules files in resources.

Reply:
Reverted last commit